Djikstra/Djikstra/Graph.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <cstdlib>
+#include <fstream>
+#include <map>
+
+
+namespace Graph
+{
+    struct Edge
+    {
+        int source_id;
+        int target_id;
+        float distance = std::numeric_limits<float>::infinity();
+    };
+
+    struct Vertex
+    {
+        std::string name;
+        std::vector <Edge> edges;
+        int parent_id = -1;
+        bool visited = 0;
+        float distance = std::numeric_limits<float>::infinity();
+    };
+
+    class Graph
+    {
+    public:
+        std::vector <Vertex> m_vertex;
```

Comment:
This should be `private`. [Don't exhibit too much.](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming))